confChamber.py

```diff
@@ -93,6 +97,13 @@
             print 'Set link %d VFAT%d VThreshold1 to %i'%(options.gtx,event.vfatN,event.vt1+options.vt1bump)
             writeVFAT(ohboard, options.gtx, int(event.vfatN), "VThreshold1", int(event.vt1+options.vt1bump),0)
             writeVFAT(ohboard, options.gtx, int(event.vfatN), "ContReg3", int(event.trimRange),0)
+
+        if options.verify:
+            if options.vt1bump > 0:
```

Comment:
I think that at line 102, `vt1bump` *could* be negative (this "feature" has been used in the past)

Reply:
Thanks for this point, I will change this `if` statement to reflect this possibility.